models.py

```diff
@@ -728,7 +729,9 @@ class ApiKeyModelView(BaseOnlyUserOwnedModelView):
     def _format_qrcode(view, context, model, name):
         admin = model.account_admin if model.account_admin else False
         address = model.user.wallet_address if model.user.wallet_address else ''
-        data = 'zapm_apikey:%s?secret=%s&name=%s&admin=%r&address=%s' % (model.token, model.secret, model.name, admin, address)
+        url = urlparse(request.base_url)
+        server = '{}{}{}{}'.format(url.scheme, '://', url.netloc, '/')
```

Comment:
this is much better expressed as:
`server = '{}://{}/'.format(url.scheme, url.netloc)`

Reply:
changed it.